src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -135,8 +135,23 @@ export function createAnimatedComponent(
     static contextType = SkipEnteringContext;
     context!: React.ContextType<typeof SkipEnteringContext>;
 
+    // Copy of props with some of the values updated
+    safeProps: {
+      animatedProps: Partial<AnimatedComponentProps<AnimatedProps>>;
+    };
+
     constructor(props: AnimatedComponentProps<InitialComponentProps>) {
       super(props);
+
+      const animatedProps = this.props.animatedProps;
+
+      this.safeProps =
+        animatedProps === null || animatedProps === undefined
+          ? { animatedProps: {} }
+          : {
+              animatedProps,
+            };
```

Comment:
can we write it like that?
```suggestion
          : { animatedProps };
```